portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,16 +63,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String enteredName = request.getParameter("name-entry");
     String enteredComment = request.getParameter("comment-entry");
-    Comment comment = new Comment(enteredName, enteredComment);
-    //String json = convertToJson(comment);
-    comments.add(enteredComment); //testing to se if it's working
+    long timestamp = System.currentTimeMillis();
+    //upload via datastore
+    Entity commentEntity = new Entity("Comment");
```

Comment:
Why don't your just create your Comment Object here and store that? Assuming that comments are immutable, that would save you having to read/write these properties every time